signer/src/transaction_coordinator.rs

```diff
@@ -827,6 +857,71 @@ where
         }
     }
 
+    fn authenticate_message(
+        packet: &wsts::net::Packet,
+        public_keys: &hashbrown::HashMap<u32, p256k1::point::Point>,
+        public_key_point: p256k1::point::Point,
+        sender_is_coordinator: bool,
+    ) -> bool {
+        macro_rules! check_signer_public_key {
+            ($message:ident) => {
+                match &public_keys.get(&$message.signer_id) {
```

Comment:
Congrats on the macro 🎉. It's pretty straightforward but I don't think we should add it. In general, I think we should try hard not to add them, since they are typically harder for people to understand, debug, and so on. Plus VS code barfs when looking at them. So we've tended to not use them, and I'd like to continue that trend unless the payoff clearly exceeds the cost.

Now, this macro is a pretty simple one, and I think we can get by with making it a closure (or proper) function that takes in the `signer_id` (without the message) and goes about it's day.

Reply:
Closure seemed best, to avoid passing in packet/etc.  Fixed.